src/queries/__tests__/placeholderText.test.tsx

```diff
@@ -78,3 +78,41 @@ test('byPlaceholderText queries support hidden option', () => {
     `"Unable to find an element with placeholder: hidden"`
   );
 });
+
+test('byPlaceHolderText should return composite component if useLegacyQueries option is true', () => {
+  configure({ useLegacyQueries: true });
+  const { getByPlaceholderText } = render(
+    <TextInput placeholder="placeholder" />
+  );
+
+  expect(getByPlaceholderText('placeholder').type).toBe(TextInput);
+});
+
+test('byPlaceHolderText should return host component if useLegacyQueries option is false', () => {
+  const { getByPlaceholderText } = render(
```

Comment:
to not depend on the default option which we're not currently testing, I would specify: 
```suggestion
    configure({ useLegacyQueries: false });
  const { getByPlaceholderText } = render(
```
(same comment for the other queries tests)

Reply:
What I do now is set useLegacyQueries option to false in .breaking tests files, is that ok for you ?